src/gii/default/email-template.php

```diff
@@ -52,16 +52,32 @@ public function safeUp()
             ->select('id')
             ->where(['key' => '<?= $generator->key ?>'])
             ->scalar();
-
-        $this->insert($this->translationTableName, [
+        
+        <?php 
+           $module = Yii::$app->getModule('email-templates');
+           ?>
+            <?php if ($module): ?>
+            <?php if ($module->languageProvider): ?>
+            <?php foreach($module->languageProvider['languages'] as $language): ?>
```

Comment:
Please use language provider helper instead of getting provider form the module.

```php
foreach (\motion\i18n\helpers\LanguageHelper::getInstance()->getAll() as $languge)
```